homeassistant/components/snips.py

```diff
@@ -144,12 +173,40 @@ def snips_say_action(call):
                            json.dumps(notification))
         return
 
+    async def feedback_on(call):
+        """Turn feedback sounds on."""
+        site_ids = ([call.data.get(ATTR_SITE_ID)]
+                    if call.data.get(ATTR_SITE_ID)
+                    else config[DOMAIN].get(CONF_SITE_IDS))
+        for site_id in site_ids:
+            payload = json.dumps({'siteId': site_id})
+            hass.components.mqtt.async_publish(
+                FEEDBACK_ON_TOPIC, payload, qos=1, retain=True)
+
+    async def feedback_off(call):
+        """Turn feedback sounds off."""
+        site_ids = ([call.data.get(ATTR_SITE_ID)]
+                    if call.data.get(ATTR_SITE_ID)
+                    else config[DOMAIN].get(CONF_SITE_IDS))
+        for site_id in site_ids:
```

Comment:
Extract this into helper methods please.

Reply:
Roger